packages/plugin-request-logger/package.json

```diff
@@ -2,24 +2,25 @@
 	"name": "webstone-plugin-request-logger",
 	"version": "0.1.0",
 	"scripts": {
-		"build": "npm run clean:build && npm run build:cli && npm run build:web",
-		"build:web": "vite build && npm run package",
-		"build:cli": "node scripts/build-cli.js build && npm run copy-templates",
+		"build": "npm run clean:build && npm run cli:build && npm run web:build",
+		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
 		"clean:build": "rimraf ./dist",
-		"copy-templates": "cp -a ./src/cli/templates ./dist/cli",
-		"dev": "npm-run-all --parallel dev:cli dev:web",
-		"dev:cli": "node scripts/build-cli.js dev",
-		"dev:web": "vite dev",
-		"preview": "vite preview",
+		"cli:build": "node scripts/build-cli.js build && npm run templates:copy",
+		"cli:dev": "node scripts/build-cli.js dev",
+		"dev": "npm run clean:build && npm-run-all --parallel cli:dev web:dev templates:dev",
+		"format": "prettier --plugin-search-dir . --write .",
+		"lint": "prettier --plugin-search-dir . --check . && eslint .",
 		"package": "svelte-kit sync && svelte-package -o ./dist/web && publint",
 		"prepublishOnly": "npm run package",
-		"test": "npm run test:integration && npm run test:unit",
-		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
-		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
-		"lint": "prettier --plugin-search-dir . --check . && eslint .",
-		"format": "prettier --plugin-search-dir . --write .",
+		"preview": "vite preview",
+		"templates:copy": "cp -a ./src/cli/templates ./dist/cli",
+		"templates:dev": "nodemon --watch src/cli/templates --ext '.ejs' --exec 'npm run templates:copy'",
```

Comment:
Very clever, I like it!

In a future version of Node.js, we'll hopefully be able to use `node --watch-path` ([docs](https://nodejs.org/api/cli.html#--watch-path))